website/docs/r/storage_object_iam_policy.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "google"
+page_title: "Google: google_storage_object_iam_policy"
+sidebar_current: "docs-google--storage-object-iam-policy"
+description: |-
+ Allows management of an IAM policy for a Google Storage Bucket Object.
+---
+
+# google\_storage\-_object\_iam\_policy
+
+Allows creation and management of an IAM policy for an existing Google Storage Bucket Object.
+
+## Example Usage
+
+```hcl
+resource "google_storage_bucket" "bucket" {
+  display_name = "%s"
+}
+
+resource "google_storage_object" "object" {
+  display_name = "%s"
+	bucket = "${google_storage_bucket.bucket.name}"
+}
+
+resource "google_storage_object_iam_policy" "object" {
+  object     = "${google_storage_object.object.name}"
+  policy_data = "${data.google_iam_policy.admin.policy_data}"
+}
+
+data "google_iam_policy" "admin" {
+  binding {
+    role = "roles/editor"
+
+    members = [
+      "user:jane@example.com",
+    ]
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `object` - (Required) The Object ID.
+
+* `policy_data` - (Required) The `google_iam_policy` data source that represents
+    the IAM policy that will be applied to the project. The policy will be
+    merged with any existing policy applied to the project.
+
+    Changing this updates the policy.
+
+    Deleting this removes the policy, but leaves the original object policy
```

Comment:
Is this true? Looking up at the code, it looks like the policy is just set to an empty policy on delete.